ticketutil/servicenow.py

```diff
@@ -14,12 +14,24 @@
     logging.basicConfig(level=logging.INFO)
```

Comment:
Please remove lines 9-14 in servicenow.py. I re-worked logging in a recent commit:

https://github.com/dmranck/ticketutil/commit/17c1560b806ba567d486e0c51fa2e94d501fe00c

You can also remove import os as a result of this. But keep the blank line between 'import os' and 'import requests' as per PEP8. See top of jira.py for clarification:

https://github.com/dmranck/ticketutil/blob/master/ticketutil/jira.py#L1